app/components/Search/QuickLinks.tsx

```diff
@@ -0,0 +1,28 @@
+import ResolveLink from '../ResolveLink';
+import styles from './Search.css';
+import type { NavigationLink } from './utils';
+
+type Props = {
+  title: string;
+  links: NavigationLink[];
+  onCloseSearch: () => void;
+};
+
+const QuickLinks = ({ title, links, onCloseSearch }: Props) => {
+  return (
+    <div>
+      <h2 className={styles.quickLinksHeader}>{title}</h2>
+      <div className={styles.quickLinks}>
+        {links.map((link) => (
+          <ResolveLink
+            key={link[0]}
+            link={link}
+            {...{ className: styles.quickLink, onClick: onCloseSearch }}
```

Comment:
Why is this a spread object when its not dynamic?

Reply:
well. This is just another way of ignoring TS instead of fixing the types :grimacing:.
(most likely you need to fix the typing for the props to `ResolveLink`)